### ISSUE TITLE
A bishop 

### ISSUE THREAD

**Issue by jsaintlauren** [srophe/srophe-app-data#445]
http://syriaca.org/work/1630 should we realphabetize so that it comes under "B" instead of "a"?

**Comment by wsalesky** [comment]
This probably should be added to the browse function I think, it should ignore 'A' and 'The' in the sort function. (So a bug in the srophe-eXist-app repo instead.)

**Comment by jsaintlauren** [comment]
OK!  
Here is another example : http://wwwb.library.vanderbilt.edu/exist/apps/srophe/work/1646

**Comment by davidamichelson** [comment]
We can discuss today

**Comment by davidamichelson** [comment]
Yes please exclude A, An, The, from the browse headwords alphabetization in BHSE.
